worlds/World 1/world.mt

```diff
@@ -1 +1 @@
-gameid = default
\ No newline at end of file
+gameid = default
```

Comment:
gui_scaling_filter

Reply:
I don't know what it is. I tried to cancel them but it did not work.

Anyway it's safe, there is no change.